ddpui/celeryworkers/tasks.py

```
@@ -161,6 +163,36 @@ def clone_github_repo(
+
+        try:
+            elementary_service.create_elementary_profile(org)
+        except Exception as error:
+            taskprogress.add(
+                {
+                    "message": "elementary setup failed",
+                    "error": str(error),
+                    "status": "failed",
+                }
+            )
+            logger.exception(error)
```

Ruff (0.14.7)

Redundant exception object included in `logging.exception` call

(TRY401)